Userland/Libraries/LibC/arch/x86_64/crti.S

```diff
@@ -1,15 +1,15 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
```

Comment:
This is a bit akward as andreas didn't touch the file (at least as far as I can tell), same in `crtn.S`

Reply:
I found it weird to have 2 copyright headers for 2 lines of ASM. You're right that this is not good, will change.